addons/web/static/tests/views/fields/color_field_tests.js

```diff
@@ -100,6 +100,26 @@ QUnit.module("Fields", (hooks) => {
         assert.doesNotHaveClass(target.querySelector(".o_data_row"), "o_selected_row");
     });
 
+    QUnit.test("read-only color field in editable list view", async function (assert) {
+        await makeView({
+            type: "list",
+            serverData,
+            resModel: "partner",
+            arch: `
+                <tree editable="bottom">
+                    <field name="hex_color" readonly="1" widget="color" />
+                </tree>`,
+        });
+
+        assert.containsN(
+            target,
+            '.o_field_color input:disabled',
+            "the field should not be editable"
+        );
+
+        await click(target.querySelector(".o_field_color input"));
```

Comment:
Test seems fine, if you're going to have this click, you should check something after the click (eg that the selected color did not change), though in this case if the input wasn't disabled it would open the native color picker, which we don't want. I think the single assert that you have that checks that the input is disabled is enough

Reply:
Ok, thanks!